satellite/peer.go

```diff
@@ -181,6 +181,7 @@ func New(log *zap.Logger, full *identity.FullIdentity, db DB, config *Config) (*
 	var err error
 
 	{ // setup listener and server
+		log.Debug("Starting listener and server")
```

Comment:
Any particular reason these log lines were added? The following lines aren't actually starting them, they are just setting them up. Starting happens in a different place.

Reply:
@phutchins needed them to debug some issue and wanted to keep them for future debugging too.

Reply:
We encountered issues where the setup/startup hung due to incorrect paths in the config file.
As the satellite didnt print out at which step it was, it was very difficult to troubleshoot